warehouse/admin/templates/admin/malware/verdicts/detail.html

```diff
@@ -0,0 +1,49 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "admin/base.html" %}
+
+{% block title %}{{ verdict.id.hex }}{% endblock %}
```

Comment:
again, I'd probably drop .hex and just go with the UUID itself, perhaps change to:

{% block title %}Verdict {{ verdict.id }}{% endblock %}